package.json

```diff
@@ -22,8 +22,9 @@
   },
   "homepage": "https://github.com/debitoor/eslint-config-debitoor#readme",
   "devDependencies": {
-    "mocha": "5.2.0",
-    "mocha-eslint": "5.0.0"
+    "mocha": "9.0.1",
+    "mocha-eslint": "7.0.0",
+    "eslint": "7.29.0"
```

Comment:
I don't see any addition of tests etc.. Why is this required as a dev dependency? 👁️

Reply:
no idea, I've just update them, will remove